.github/workflows/inspektor-gadget.yml

```diff
@@ -1426,12 +1426,21 @@ jobs:
         cache: true
       id: go
     - name: Gadgets tests
+      id: gadgets-tests
```

Comment:
If you put this commit in a separated PR, we can merge it straight away.

Reply:
Already done in #3152.